### PR TITLE
Implement custom slot duration runtime api for runtimes

### PR DESCRIPTION
**Pull request by tmpolaczyk**
Technically only container chains need this, but if we don't implement it for tanssi then the code does not compile, because we assume that the ContainerChainClient has the same set of RuntimeApi as the tanssi Client

### CODE REVIEW

File: runtime/flashbox/src/lib.rs [hunk 1]
```diff
@@ -1828,6 +1828,12 @@ impl_runtime_apis! {
             TransactionPayment::length_to_fee(length)
         }
     }
+
```
Comment (girazoki) [inline runtime/flashbox/src/lib.rs:1831]: AFAIK we can now remove the aura-runtime-api and all its requirements from our runtime with this

Comment (girazoki) [reply runtime/flashbox/src/lib.rs:1831]: but we can leave it for a separate PR

**Comment by girazoki** [comment]
I tested using this client against runtime 500 containers, and works producing every 12secs. After the upgrade to runtime600, they produce every 6 seconds without any downtime. Good job @tmpolaczyk

**Review by girazoki — APPROVED** [review]

File: runtime/flashbox/src/lib.rs (hunk 1 above)
Comment (tmpolaczyk) [reply runtime/flashbox/src/lib.rs:1831]: I prefer not to

**Pull request merged**